tracetest/1.0.0/README.md

```diff
@@ -0,0 +1,172 @@
+# Running Tracetest with Keptn
+ 
+[Tracetest](https://tracetest.io/) is a testing tool based on OpenTelemetry that permits you to test your distributed application. It allows you to use your trace data generated on your OpenTelemetry tools to check and assert if your application has the desired behavior defined by your test definitions.
+
+[Keptn](https://keptn.sh/) is a powerful tool to automate the lifecycle of your application running on Kubernetes. One of the tasks that we can do with `keptn` is to test an application and see if it is healthy and ready to be used by your users.
+
+By using the Keptn [Job Executor Service](https://github.com/keptn-contrib/job-executor-service) plugin, we can upload a Tracetest test definition and a CLI configuration to a service and run a test using the following job:
+
+```yaml
+apiVersion: v2
+actions:
+- name: "Run tracetest on your service"
+  events:
+    - name: "sh.keptn.event.test.triggered"
+  tasks:
+    - name: "Run tracetest"
+      files:
+        - data/test-definition.yaml
+        - data/tracetest-cli-config.yaml
+      image: "kubeshop/tracetest:latest"
+      cmd:
+        - tracetest
+      args:
+        - --config
+        - /keptn/data/tracetest-cli-config.yaml
+        - test
+        - run
+        - --definition
+        - /keptn/data/test-definition.yaml
+        - --wait-for-result
+
+```
+
+## Quickstart
+
+Here we will show how to use Tracetest to run these tests and help in your delivery and testing workflows using the [Pokeshop](https://docs.tracetest.io/pokeshop/) example, available at `http://demo-pokemon-api.demo` in a Kubernetes cluster.
+
+### Prerequisites
+
+In your Kubernetes cluster you should have:
+
+1. `Keptn 1.0.x` [installed](https://keptn.sh/docs/1.0.x/install/)
```

Comment:
We have now released Keptn 1.1.0.  It would be nice if Tracetest worked with that and I think it should.

Reply:
It works! I'm updating this doc, thanks!